pyproject.toml

```diff
@@ -99,11 +95,35 @@ dev = [
 
 [tool.setuptools]
 include-package-data = true
+license-files = ['LICENSE']
 
 [tool.setuptools.packages.find]
 include = ['deepecho', 'deepecho.*']
 namespaces = false
 
+[tool.setuptools.package-data]
+"*" = [
+    '*.txt',
+    '*.md',
+    '*.rst',
+    'README.md',
+    'docs/*',
+    'Makefile',
+    'make.bat',
+    '*.jpg',
+    '*.png',
+    '*.gif',
+]
```

Comment:
why are we adding this now?

Reply:
- This specifies, what should be included in the whl/tar.gz file (when making the package). We should keep this as minimal as possible (to avoid making our released package too large). 
- One thought is we should not include the `docs` folder.

Reply:
Do we need the *.txt? It seems like our MANIFEST doesn't include it anyway

Reply:
With pyproject.toml, you can actually remove MANIFEST.in file. 
> All files specified by the package-data and data-files configuration parameters in pyproject.toml and/or equivalent in setup.cfg/setup.py;
https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html

Reply:
@R-Palazzo will approve once MANIFEST.in is removed.